src/main/java/com/lightstep/opencensus/exporter/LightStepExporterHandler.java

```diff
@@ -168,13 +169,16 @@ private long fromByteArray(byte[] bytes) {
     return ByteBuffer.wrap(bytes).getLong();
   }
 
-  private long traceIdToLong(final TraceId traceId) {
+  long traceIdToLong(final TraceId traceId) {
     if (traceId == null) {
       return 0L;
     }
-    // Attempt to minimise allocations, since SpanId#getBytes currently creates a defensive copy:
+
+    // Attempt to minimise allocations, since TraceId#getBytes currently creates a defensive copy:
```

Comment:
since we're copying it below anyway, should we just get using TraceId#getBytes now?

Reply:
done